lib/vsc/utils/generaloption.py

```diff
@@ -470,6 +496,15 @@ def __init__(self, *args, **kwargs):
 
         self.environment_arguments = None
         self.commandline_arguments = None
+        self.orig_rargs = None  # copy of the original arguments
+
+    def parse_args(self, args=None, values=None):
+        """Keep a copy of the original arguments for callback / option processing"""
+        # seems like self._get_args returns a copy, but better safe then sorry here
+        self.orig_rargs = self._get_args(args)[:]
```

Comment:
well, close, `super` doesn't work because `OptionParser` doesn't derive from `object`

all tests pass with https://github.com/stdweird/vsc-base/pull/9